modules/bitgo/test/v2/unit/coins/near.ts

```diff
@@ -125,4 +126,199 @@ describe('NEAR:', function () {
       isTransactionVerified.should.equal(true);
     });
   });
+
+  describe('Explain Transactions:', () => {
+    const amount = '1000000';
+    const gas = '125000000000000';
+
+    it('should explain an unsigned transfer transaction', async function () {
+      const explainedTransaction = await basecoin.explainTransaction({
+        txPrebuild: {
+          txHex: rawTx.transfer.signed,
+        },
+      });
```

Comment:
What happen if we add
feeInfo: {
          fee: '5000',
        },

to the parameters? it is allowed by the function, can we add a test with that?